### ISSUE TITLE
Missing Metadata

### ISSUE THREAD

**Issue by dyhan316** [ins-amu/virtual_aging_brain#1]

**Comment by dyhan316** [comment]
Hello! I was referred to this code from Spase and have encountered a problem regarding the meta-data required for running the first `.ipynb` file.

As outlined in `Data alert` in the `01_vab_github_sc_investigation.ipynb` file, I have downloaded (some) of the EBRAINS dataset, as below (I have only downloaded 5 subjects): 
![image](https://user-images.githubusercontent.com/34816251/170218499-4c059d86-2b45-48b2-a78e-80ff38a55775.png)

I have encountered two problems:
1. When defining `subjs`, the error `[Errno 2] No such file or directory: '~/virtual_aging_brain/data/external/Julich'` pops up. To resolve this I have tried the following two things:
* first, the `~/data` folder didn't exist in the git version of the code, I made the `data/external` folders. 
* However, I realized that the name of the folder `Julich` that is required by the code does not match that of the EBRAINS dataset (`Julich-Brain` (as the picture above shows)

2. When defining `parce_list`, The error,  `[Errno 2] No such file or directory: '/home/dyhan316/virtual_aging_brain/data/external/Schaefer2018_100Parcels_7Networks_tab.txt'` pops up. However, as the picture above of the EBRAINS dataset I downloaded shows, there was no file named `Schaefer2018_100Parcels_7Networks_tab.txt'`. 

To summarize, my questions are as follows:

1. Is the `~/data/external` folder supposed to be made by me? 
2. What should I set as the `~/virtual_aging_brain/data/external/Julich` folder? The folder name in the EBRAINS dataset (`Julich-Brain/Streamlines` does not match the name that is required by the program)
3. Where can I obtain the missing `Schaefer2018_100Parcels_7Networks_tab.txt` file? 

Thank you! 

p.s. your paper was very interesting and a joy to read!

Best,

Danny

**Comment by dyhan316** [comment]
Also, it might also be worth noting that while the `Data alert` block says `dataset is located in the folder $MYPATH/aging_virtual_brain/data/external`, when downloading from github the name of the folder is `virtual_aging_brain`, not `aging_virtual_brain`

**Comment by i-Zaak** [comment]
Dear Danny,
 thank you for your interest in the paper and the codes. Unfortunately, the 1000BRAINS dataset is not completely shared on EBRAINS yet (e.g. functional data is missing), and also some of its organization changed in the process of curation. The codes here refer mostly to the previous version of the dataset.

There is an updated version of the code available in the [Showcase 1](https://wiki.ebrains.eu/bin/view/Collabs/sga3-d1-2-showcase-1), which works with the dataset available in EBRAINS, of course with the caveat that the latter steps using empirical functional data require access to private data supplement.

Nevertheless, I have uploaded at least the missing parcellation files in #4 

Hope this helps
Jan

**Comment by dyhan316** [comment]
Dear Jan,

Thank you for the detailed response! 

Ill be sure to take a look at the updated code you referred to! Also thank you for the parcellation file. :) 

I have one quick question if you dont mind : 
it is my understanding that TVB requires SC matrix data to work. However, the EBRAINS data that I downloaded had both tractography and SC matrix files. Are the tractography files needed elsewhere or is it just extra data that is not used in TVB? 

Thank you:)

Best Regards, 

Danny





Sent from Naver Mail app.
-----Original Message-----

**Comment by i-Zaak** [comment]
Yes, it is only the SC which is used to create a TVB model. The showcase is using a [simple wrapper](https://gitlab.ebrains.eu/fousekjan/tvb-ebrains-data/-/blob/master/notebooks/demo.ipynb) to pull the data from KG on demand, but downloading them locally works just fine.

**Comment by dyhan316** [comment]
Dear Jan
 
Ah! I see :) That clears things up a bit
 
Thank you!
 
Danny
-----Original Message-----
